src/main/java/io/vertx/core/http/HttpServerRequest.java

```diff
@@ -170,6 +169,18 @@ default String getParam(String paramName) {
     return params().get(paramName);
   }
 
+  /**
+   * Return the first param value or defaultValue if the query param is not provided.
```

Comment:
"Return the first param value with the specified name or `@code{defaultValue}` when the query param is not present."

Reply:
Done.